core/rio/jsonld/src/main/java/org/eclipse/rdf4j/rio/ndjsonld/NDJSONLDParserFactory.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
```

Comment:
In all of the files you added this needs to be present and be the current text/year (see the guideline).